data/json/recipes/other/other.json

```diff
@@ -40,6 +40,19 @@
     "tools": [ [ [ "drawing_tool", 5, "LIST" ] ] ],
     "components": [ [ [ "cardboard", 10 ] ] ]
   },
+  {
+    "result": "outfit_storage",
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "time": "1 s",
+    "autolearn": true,
+    "reversible": true,
+    "//": "Tying equipment or other fasteners. Maybe one day we will have coat hangars.",
+    "components": [ [ [ "cordage", 10, "LIST" ] ] ]
```

Comment:
Shouldn't this be free? An outfit is just an arbitrary set of cloth, you don't need cordage to look at your cloth and decide they go well together.

Reply:
You end up getting the cost back anyway once you use it.

But I thought it would be bad if the player could summon "huge items"(100L) by itself without spending anything.

Reply:
I feel like an outifit should be a UI thing, like an entry in a dedicated menu, or a some symbols next to the cloth, rather than an ingame item with volume and weight that you need component to craft.

Reply:
I'm thinking of a refactor to use actual pockets so that the volume and weight can be generated dynamically and it would serve as more of a useful wrapper than anything else. Need to do some trickery to make sure players can't put all their loot into the endlessly expanding pocket though.

Reply:
Uses pockets now so I think this is resolved.